apps/email-otp-authentication-portal/src/main/webapp/js/scripts.js

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Shall we fix the indentation of this file? And I have noticed the same on other scripts.js files as well